client/displayservers/Wayland/shell_libdecor.c

```diff
@@ -149,3 +160,17 @@ void waylandMinimize(void)
 {
   libdecor_frame_set_minimized(wlWm.libdecorFrame);
 }
+
+void waylandShellResize(int w, int h)
```

Comment:
A stub needs to be implemented for this in `shell_xdg.c`.

Reply:
Added.